client.lua

```diff
@@ -1,3 +1,21 @@
+FRAMEWORK = nil
```

Comment:
This block of code is held globally within `client_util.lua`, BUT i think it's loaded afterwards -> https://github.com/kylemcshea/dream-postal/blob/master/fxmanifest.lua#L18

So we can go with 1 of 2 routes
1. Delete this block of code and within `fxmanifest.lua` we have `client_util.lua` loaded up before `client.lua`
```lua
  'client_util.lua',
  'client.lua',
```

2. we delete this block of code from `client_util.lua` and leave everything as is. I'm down for either. 

I'll leave the decision to you :) we just dont want duplicate blocks of code is the only thing we are looking to move from